### ISSUE TITLE
Troubleshooting when no sound comes

### ISSUE THREAD

**Issue by FeralFox** [duncanthrax/scream#173]
Hello,

I have a problem and I would be very happy if you can give me a hint how I can troubleshoot it.

I use Scream on Windows 10. Scream Receiver is on (command line) Debian on a Raspberry. This setup has worked but "suddenly" stopped working. "Suddenly" is probably after an update of some libraries

Here is what I tried:
* Installed new version (3.9) on Windows
* Installed new Receiver (3.9) on Linux
* Tested with nvlc that Linux can play sounds (it can)
* "scream -v" on Linux shows me "Pulseaudio output \nStarting multicast receiver"
* Rebooted everything
* When I start scream, the speakers make a small noise, so I guess speakers are enabled when scream starts.
* When I set audio output to scream on Windows, I get an upload. So scream on Windows is streaming.

**Comment by danleyb2** [comment]
switching my speaker configuration to 3.1 was the fix for me
https://github.com/duncanthrax/scream#setting-up-default-speaker-configuration-optional

**Comment by FeralFox** [comment]
Hi danleyb2,

thank you for the hint! Unfortunately, that didn't work for me

**Comment by FeralFox** [comment]
@danleyb2 
Yes this is what I do too usually. I also experimented with output types (alsa, pulse) and the interface (I used the IP address of the Pi in my local network there, so 192.168.0.3)
In the meantime I also switched the Raspberry and re-installed Raspian, but still no luck.

**Comment by coxtor** [comment]
I struggled with connecting a windows 11 client too. I managed to fix it by checking whether multicast was working at by checking if I could connect to a local multicast source - which it diddn't . 
To fix the issue I needed to disable automatic metric in my windows network adapter  in the adapter settings under tcp -> advanced -> uncheck automatic metric and set the value to "1".
I followed this guide:
https://windowsreport.com/firewall-blocking-multicast/

Perhaps this should be added to the readme.
Hope I could help.
